cmd/kubeadm/app/util/pkiutil/pki_helpers.go

```diff
@@ -218,7 +218,7 @@ func WriteCSR(csrDir, name string, csr *x509.CertificateRequest) error {
 		return errors.Wrapf(err, "failed to make directory %s", filepath.Dir(csrPath))
 	}
 
-	if err := os.WriteFile(csrPath, EncodeCSRPEM(csr), os.FileMode(0600)); err != nil {
+	if err := os.WriteFile(csrPath, EncodeCSRPEM(csr), os.FileMode(0600) | os.FileMode(syscall.S_IWUSR)); err != nil {
```

Comment:
> revoke world-accessible permissions that are unnecessary to make the file give the written permission to the owner of a file.

the permission used to be 644, but now it's 600.
https://github.com/kubernetes/kubernetes/issues/81116

600 already means RW only for the owner, which in kubeadm's case is root.

Reply:
it make less error than 600